cppForSwig/libbtc/src/secp256k1/src/tests.c

```diff
@@ -1,4525 +0,0 @@
-/**********************************************************************
```

Comment:
Is it necessary to delete a test suite?

Reply:
I actually only copied libbtc subdir from the upstream goapig/BitcoinArmory dev branch, so it's likely deleted there (I don't see it in my cloned copy)